test/README.md

```diff
@@ -331,6 +334,29 @@ err = WaitForTaskRunState(c, hwTaskRunName, func(tr *v1alpha1.TaskRun) (bool, er
 _[Metrics will be emitted](https://github.com/knative/pkg/tree/master/test#emit-metrics)
 for these `Wait` methods tracking how long test poll for._
 
+## Conformance tests
+
+### Setup
+
+Environment variables used by conformance tests:
+
+- `KO_DOCKER_REPO` - Set this to an image registry your tests can push images to
```

Comment:
I'm not sure this is needed when running tests with `go test`, since it should only be needed when you're building and pushing images. If it's not, can you remove this?

Reply:
You're right. There is no need to set KO_DOCKER_REPO.